usmqe/ceph/commands.py

```diff
@@ -10,6 +13,11 @@
 
 
 LOGGER = pytest.get_logger('usmceph.commands', module=True)
+usmqe.usmssh.KEYFILE = "~/.ssh/id_rsa"
+try:
+    usmqe.usmssh.KEYFILE = pytest.config.getini("usm_ssh_keyfile")
+except ValueError:
+    pass
```

Comment:
I agree with @mbukatov, it would be better to completely remove the KEYFILE option and rely on proper global SSH configuration.